### PR TITLE
Add nova multiplecells nodeset templates

### PR DESCRIPTION
**Pull request by jamepark4**
Currently working with [1] to allow for deployment of two nodesets with each nodeset corresponding with a respective cell.

[1] https://github.com/openstack-k8s-operators/architecture/pull/401

### CODE REVIEW

**Review by lewisdenny — CHANGES_REQUESTED** [review]
lgtm, to get the docs ci job to pass you need to rebase to catch: https://github.com/openstack-k8s-operators/ci-framework/pull/2526

File: roles/ci_gen_kustomize_values/templates/nova-three-cells/edpm-nodeset2-values/values.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+{% set _original_nodeset = (original_content.data | default({})).nodeset | default({}) %}
+{% set _original_nodes = _original_nodeset.nodes | default({})                         %}
+{% set _original_services = _original_nodeset['services'] | default([])                %}
+{% for _inst in cifmw_baremetal_hosts.keys()                                           %}
+{%     if _inst.startswith('compute')                                                  %}
+{%         set _ = instances_names.append(_inst)                                       %}
+{%     endif                                                                           %}
+{% endfor                                                                              %}
+{% set inst_stop_idx = (instances_names | length) // 2                                 %}
+{% set nodset_two_instances = instances_names[inst_stop_idx:]                          %}
```
[the remark this answers is not in the document]
Comment (jamepark4) [reply roles/ci_gen_kustomize_values/templates/nova-three-cells/edpm-nodeset2-values/values.yaml.j2:14]: ack thankyou

File: roles/ci_gen_kustomize_values/templates/nova-three-cells/edpm-nodeset-values/values.yaml.j2 [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+{% set _original_nodeset = (original_content.data | default({})).nodeset | default({}) %}
+{% set _original_nodes = _original_nodeset.nodes | default({})                         %}
+{% set _original_services = _original_nodeset['services'] | default([])                %}
+{% for _inst in cifmw_baremetal_hosts.keys()                                           %}
+{%     if _inst.startswith('compute')                                                  %}
+{%         set _ = instances_names.append(_inst)                                       %}
+{%     endif                                                                           %}
+{% endfor                                                                              %}
+{% set inst_stop_idx = (instances_names | length) // 2                                 %}
+{% set nodset_one_instances = instances_names[:inst_stop_idx]                          %}
```
[the remark this answers is not in the document]
Comment (jamepark4) [reply roles/ci_gen_kustomize_values/templates/nova-three-cells/edpm-nodeset-values/values.yaml.j2:14]: ack thank you

**Review by hjensas — COMMENTED** [review]
/lgtm

**Review by lewisdenny — APPROVED** [review]
/approve

**Pull request merged**